airflow/contrib/executors/kubernetes_executor.py

```diff
@@ -139,7 +139,7 @@ def __init__(self):
 
         # NOTE: user can build the dags into the docker image directly,
         # this will set to True if so
-        self.dags_in_image = conf.get(self.kubernetes_section, 'dags_in_image')
+        self.dags_in_image = conf.getboolean(self.kubernetes_section, 'dags_in_image')
```

Comment:
This _might_ be considered a breaking change.  It appears that `getboolean` throws an exception if the value is empty, so if someone does not update their config file to include `dags_in_image` their setup will break.

Looking for suggestions on how best to work around this.